modules/programs/zsh.nix

```diff
@@ -483,7 +483,7 @@ in
         ''}
 
         ${optionalString cfg.prezto.enable
-            (builtins.readFile "${pkgs.zsh-prezto}/runcoms/zshrc")}
+            (builtins.readFile "${pkgs.zsh-prezto}/share/zsh-prezto/runcoms/zshrc")}
```

Comment:
Instead of using `readFile`, why not source the file directly? `"source ${pkgs.zsh-prezto}/share/zsh-presto/runcoms/zshrc"`

Sourcing works for file not marked as executable.

Reply:
The runcom itself is a tiny file, amounting to a few comments (which can be helpful to see when examining your zshrc) and a single source of zsh-prezto's entrypoint anyway, so I'd prefer readFile in this specific instance.